src/Idris/Package.idr

```diff
@@ -795,6 +795,25 @@ errorMsg = unlines
   , "    --build-dir <dir>"
   , "    --output-dir <dir>"
   ]
+{-
```

Comment:
Left in by accident?

Reply:
See issue #2087 

I decided to keep the multiline version.  Once #2087 is fixed, it can then be used in place of the 'unlines' version. If it is not acceptable I will delete it.

Reply:
I think we should remove the comment in the meantime. Until the multiline version is actually made to be the version in-code, it creates an extra place where any modifications to the message needs to be made and for the same reason the commented out version will need to be checked for consistency after the bug is fixed so it doesn't really save time at that point IMO.

Reply:
Done.  Thanks for the review!